### DISCUSSION TITLE
List of ideas for things we can do

### DISCUSSION THREAD

**Discussion by Joseph-Rance** [Joseph-Rance/Computing-Soc#9] [Ideas]
- Python Unit tests
 - Rust tutorial
 - Unix talk

**Comment by kashparty** [comment]
- How programming languages are made (maybe talk about M?)
- That AI tournament thing we were going to do
- Presenting somebody's NEA
- Programming competition to see who's gonna be the next leader of comsoc
- Google HashCode preparation?

**Comment by Joseph-Rance** [comment]
- Coding a physics simulation
 - Coding Boyds
 - Dynamic programming example
